openlibrary/plugins/upstream/tests/test_account.py

```diff
@@ -1,7 +1,20 @@
+# -*- coding: utf-8 -*-
+
 from .. import account
 import web
+import os
 import re
 import pytest
+import sys
```

Comment:
Just FYI, isort will have a different sort order but we have no rules about this yet.